client/components/payment-confirm-illustration/index.tsx

```diff
@@ -10,7 +11,10 @@ import classNames from 'classnames';
  */
 import './styles.scss';
 
-const PaymentConfirmIllustration = ( { hasBorder, Icon = () => null } ) => {
+const PaymentConfirmIllustration: React.FunctionComponent< {
```

Comment:
It would be great to introduce a test for this component (a snapshot will work as we have nothing yet).

Reply:
Added tests in https://github.com/Automattic/woocommerce-payments/pull/3910/commits/eb2f8fb875423f9788085111638972effb3a6f41